content/static/js/custom.cfr.js

```diff
@@ -17,4 +20,64 @@ function checkNavbar() {
   } else {
     showNavbar();
   }
-};
\ No newline at end of file
+};
+
+var dummy, dummy2, dummyJSON, txt, x, xmlhttp;
+
+function submitCFR() {
+  dummy = { name1: document.getElementById("cfrName").value, num1: document.getElementById("cfrNum1").value,
+  num2: document.getElementById("cfrNum2").value, num3: document.getElementById("cfrNum3").value};
+  dummyJSON = JSON.stringify(dummy);
+  localStorage.setItem("dummyJSONtest", dummyJSON);
+  xmlhttp = new XMLHttpRequest();
+
+  if (testing == "true"){
+    dummy2 = JSON.parse(dummyJSON);
+    txt = "<i>added " + dummy.name1 + " " + dummy.num1 + " " + dummy.num2 + " " + dummy.num3 + " to CFR</i>";
+    x = document.createElement("P");
+    x.innerHTML = txt;
+    document.getElementById("cfrOutput").appendChild(x);
+  }
+  
+  if (simData == "true"){
```

Comment:
[NON-ESSENTIAL] I don't really understand what this code is doing (please comment more!). If we're adding the new row to the end of the table, consider moving it to execute after the request returns like I mentioned with the previous bit.

Reply:
@angelak1 Hey could you take a look at this? I think you wrote this section and I don't quite understand what it's doing either.